### ISSUE TITLE
Error: strange here

### ISSUE THREAD

**Issue by dereneaton** [cecileane/iBPP#1]
Hi Cecile, 

I'm running a ibpp analysis and got the following error:
```
Error: strange here
```

I saw this once before too. On my latest analysis I was running without data (useseqdata=0, usetraitdata=0), which may be relevant. I'm also curious that the delimitation didn't change over the course of this analysis. Should I be changing some other prior having to do with the node-collapsing/reversible jump frequency? Thanks in advance for your help!

stdout progress before crash:
```
...
  5% 0.69 0.30 0.29 0.27 0.33 1.00  24 0.0000 10000  0.01214 17:28
 10% 0.69 0.33 0.26 0.30 0.33 1.00  24 0.0001 10000  0.01006 26:15
 15% 0.68 0.30 0.25 0.40 0.33 1.00  24 0.0001 10000  0.00723 34:55
 20% 0.68 0.29 0.23 0.46 0.33 1.00  24 0.0001 10000  0.00689 43:34
 25% 0.68 0.28 0.23 0.49 0.33 1.00  24 0.0001 10000  0.00666 52:13
 30% 0.68 0.27 0.22 0.52 0.33 1.00  24 0.0001 10000  0.00717 1:00:49
 35% 0.68 0.26 0.23 0.51 0.33 1.00  24 0.0001 10000  0.00871 1:09:22
 40% 0.68 0.26 0.23 0.51 0.33 1.00  24 0.0001 10000  0.00898 1:17:57
 45% 0.68 0.25 0.24 0.51 0.33 1.00  24 0.0000 10000  0.00858 1:26:33
 50% 0.68 0.26 0.24 0.50 0.33 1.00  24 0.0001 10000  0.00787 1:35:19
 55% 0.68 0.27 0.23 0.50 0.33 1.00  24 0.0001 10000  0.00723 1:44:04
 60% 0.68 0.26 0.23 0.51 0.33 1.00  24 0.0001 10000  0.00673 1:52:45

Error: strange here.
```

the ctl file
```
seed = 195210
seqfile = /ysm-gpfs/home/de243/Canarium-GBS/analysis_bpp/example2.ibpp.seq.txt
Imapfile = /ysm-gpfs/home/de243/Canarium-GBS/analysis_bpp/example2.ibpp.imap.txt
mcmcfile = /ysm-gpfs/home/de243/Canarium-GBS/analysis_bpp/example2.ibpp.mcmc.txt
outfile = /ysm-gpfs/home/de243/Canarium-GBS/analysis_bpp/example2.ibpp.out.txt
traitfile = /ysm-gpfs/home/de243/Canarium-GBS/analysis_bpp/example2.ibpp.traits.txt
nloci = 200
cleandata = 0
speciesdelimitation = 1 1 1 1
ntraits = 27
nindT = 36
usetraitdata = 0
useseqdata = 0
nu0 = 0
kappa0 = 0
species&tree = 6 A B C D E F
                 9 4 3 6 2 13
                 ((((D,C),B),(E,F)),A);
thetaprior = 2 200
tauprior = 2 200 1
finetune = 1: 100.0 0.0001 0.0001 0.0001 0.1 0.001 0.1 0.1
print = 1 0 0 0
burnin = 10000
sampfreq = 2
nsample = 100000
```

**Comment by cecileane** [comment]
The error is strange indeed: it's during the rjMCMC proposal that attempts to split a current species (an internal node in the guide tree). I have not seen the error before, so I'm really not sure.

The fact that the sampled delimitation does not change is definitely not good: without using the data, the MCMC should sample all delimitations according to the prior, and not stay stuck to a single delimitation. That's probably bad mixing, from tuning parameters that aren't adapted to the prior. 

With a Γ(2,200) prior for the age τ of the group, that's an average of 2/200=0.01 substitutions/site. So the root of gene trees should have ages around 0.01. The "finetune" parameter (step length) to adjust the node ages in the gene trees is the first number after `finetune = 1:`, so it's 100.0 in your output above. This parameter seems off compared to values that are of the order of 0.01 or less (for non-root nodes). Did you try to change this 100.0 to something more like 0.001, and see if that might be enough to provide a bit of mixing for this analysis?

**Comment by dereneaton** [comment]
Thanks for you reply Cecile. I chose the first tuning parameter to be close to the value that it ended up auto-tuning to when I used only sequence data in BPP 3.1. It seemed to consistently favor very large values. But you're right, when I lowered the first fine-tune value in ibpp it seems to be running without raising the 'strange' error. I'll play with the params more and see if I can get the chain to mix better. Thanks!
